readme.md

```diff
@@ -93,6 +93,39 @@ prompt pure
 | **`PURE_GIT_DOWN_ARROW`**        | Defines the git down arrow symbol.                                                             | `⇣`            |
 | **`PURE_GIT_UP_ARROW`**          | Defines the git up arrow symbol.                                                               | `⇡`            |
 
+
+## Colors
+
+Colors can be changed by using [`zstyle`](http://zsh.sourceforge.net/Doc/Release/Zsh-Modules.html#The-zsh_002fzutil-Module) with a pattern of the form `:prompt:pure:$color_name` and style `color`. The color names, their default, and what part they affect are:
```

Comment:
Can you link to somewhere that clearly lists all the supported named colors and what the `242` numbered colors refer to?

Reply:
Ok, I found this:

> Support for 24-bit true color terminals has been added. Hex triplets can be used when specifying colours for prompts and line editor highlighting. On 88 and 256 colour terminals, a new zsh/nearcolor module allows colours specified with hex triplets to be matched against the nearest available colour. - http://zsh.sourceforge.net/releases.html

We should definitely load `zsh/nearcolor` and document how to specify HEX colors.

Reply:
Oh yeah, investigating `zsh/nearcolor` has been on my todo list, would be great to add support for it. Maybe we also need to add a warning if using hex colors and ZSH < 5.7.

Reply:
Support and detection of 24bit color in terminal is a shitshow at the moment, take a look [here](https://gist.github.com/XVilka/8346728) to account for it yourself.
The correct way would be test 24bit support with `echoti RGB`, but it's only supported since the latest ncurses [release](https://www.gnu.org/software/ncurses/), it isn't present in debian stable for example and a lot of application aren't yet supporting it. Testing support with `$COLORTERM`, as written in ZSH's [manual](http://zsh.sourceforge.net/Doc/Release/Zsh-Modules.html#The-zsh_002fnearcolor-Module), isn't a good idea as it can be first glance as explained [here](https://invisible-island.net/ncurses/ncurses.faq.html#no_colorterm) and [there](https://gist.github.com/XVilka/8346728#gistcomment-2249677).

In my case, I'm using most of the time `tmux` in `alacritty`, using `echoti` would indicate that my terminal can't use 24bit colors, even though I'm currently using them. Which would activate `zsh/nearcolor` and screw up my color palette.
So my stance on this is to let the user load `zsh/nearcolor` by herself, handling the corner case by herself and adding it when the terminal ecosystem will have matured enough on this feature.

Reply:
I've been thinking about that, and I'm really happy you did the research! From what you've found, it's quite clear to me that we *don't* want to automatically load `zsh/nearcolor`. I see no way around it. All we can do is inform our users, which you've already done in your last commit, muy bien 👌.

Reply:
Alright, but we should at least document how to achieve it.